assets/javascripts/discourse/components/discourse-post-event/invitees.gjs

```diff
@@ -22,57 +21,31 @@ export default class DiscoursePostEventInvitees extends Component {
     });
   }
 
-  get statsInfo() {
-    const stats = [];
-    const visibleStats = this.siteSettings.event_participation_buttons
-      .split("|")
-      .filter(Boolean);
-
-    if (this.args.event.isPrivate) {
-      visibleStats.push("invited");
-    }
-
-    visibleStats.forEach((button) => {
-      const localeKey = button.replace(" ", "_");
-      if (button === "not_going") {
-        button = "notGoing";
-      }
-
-      const count = this.args.event.stats[button] || 0;
+  get hasAttendees() {
+    return this.args.event.stats["going"] > 0;
+  }
 
-      const label = i18n(
-        `discourse_post_event.models.invitee.status.${localeKey}_count`,
-        { count }
-      );
+  get statsInfo() {
+    return this.args.event.stats["going"];
+  }
 
-      stats.push({
-        class: `event-status-${localeKey}`,
-        label,
-      });
+  get inviteesTitle() {
+    return i18n("discourse_post_event.models.invitee.status.going_count", {
+      count: this.args.event.stats["going"],
     });
-
-    return stats;
   }
 
   <template>
     {{#unless @event.minimal}}
       {{#if @event.shouldDisplayInvitees}}
         <section class="event__section event-invitees">
-          <div class="header">
-            <div class="event-invitees-status">
-              {{#each this.statsInfo as |info|}}
-                <span class={{info.class}}>{{info.label}}</span>
-              {{/each}}
-            </div>
-
-            <DButton
-              class="show-all btn-small"
-              @label="discourse_post_event.show_all"
-              @action={{this.showAllInvitees}}
-            />
```

Comment:
If this removal was intentional then `showAllInvitees` method should be deleted too

Reply:
Yes this was intentional, this button is now avail in the drop down ellipsis menu.

Reply:
will remove the method.

Reply:
now the component can be reduced to a template-only component, both service injections are unused and the properties could be inlined in the template 😃

Reply:
Would this still remain a `gjs` file? and only include <template> while still using the imports like:

```
import icon from "discourse-common/helpers/d-icon";
import i18n from "discourse-common/helpers/i18n";
import Invitee from "./invitee";
```

Reply:
yup - still a gjs but without a js class:

```gjs
const DiscoursePostEventInvitees = <template>
  ...
</template>

export default DiscoursePostEventInvitees;
```

(btw, the new i18n import is: `import { i18n } from "discourse-i18n";`)